packages/rum/README.md

```diff
@@ -141,6 +141,7 @@ The following parameters are available:
 | `silentMultipleInit`    | Boolean | No       | `false`         | Initialization fails silently if Datadog's RUM is already initialized on the page.                       |
 | `proxyHost`             | String  | No       |                 | Optional proxy host (ex: www.proxy.com), see the full [proxy setup guide][7] for more information.       |
 | `allowedTracingOrigins` | List    | No       |                 | A list of request origins used to inject tracing headers.                                                |
+| `allowLocalFile`        | Boolean | No       |                 | Allow initialization even when served from a `file:` url.                                                |
```

Comment:
Since cookies are not handled by browsers on `file` protocol and cookies are required by RUM to correctly handled session, I am not in favor of integrating this option as is for RUM.
I'd be in favor of limiting this PR for logs and we should come with a proper strategy to handle RUM sessions on `file` protocol.
Would that work for you?

Reply:
Since we support using logs without cookies, it could be easier to simply remove `checkIsNotLocalFile` in `logs.entry` instead of introducing a new option.

+1 to keep RUM as it is for now, and discuss how to use RUM without cookie further.

Reply:
I have updated the PR with these suggestions. We were only using logs, so this would definitely work for us.